src/libfmt_macros/lib.rs

```diff
@@ -150,12 +152,20 @@ pub struct Parser<'a> {
     pub errors: Vec<ParseError>,
     /// Current position of implicit positional argument pointer
     curarg: usize,
+    /// The style of the string (raw or not), used to position spans correctly
+    style: syntax::ast::StrStyle,
```

Comment:
If we made this an `Option<usize>` containing the potential offset, we don't need the libsyntax dependency. We don't really care that it's a raw string, we just need some span offset for diagnostics.